bindings/pyroot_experimental/PyROOT/test/tcomplex_radd_rsub.py

```diff
@@ -0,0 +1,43 @@
+import unittest
+import ROOT
+from ROOT import TComplex
+
+class TestTComplexMethods(unittest.TestCase):
+        """
+        Test for the operators of TComplex:
+        __add__, __sub__, __radd__, __rsub__.
+        """
+
+        c1 = TComplex(4.,0)
+        c2 = TComplex(1.,1)
+
+        d = 2.
+
+        # check if both __add__ and __radd__ give the expected result with complex + double and double + complex and if __radd__ is equal to __add__ both for complex + complex and complex + double (real components)
+        def test_radd(self):
+                self.assertEqual((self.c1 + self.d).Re(), 6.0)
```

Comment:
This one is strictly not necessary because it does not test `__radd__` (it works without the pythonisation you added, doesn't it?).